tests/e2e/testdata/scenarios/README.md

```diff
@@ -35,4 +35,6 @@ a top-level field `TEST` on the object:
 
 * Setting `TEST: WAIT-FOR-HTTP-REQUEST`along with `VALUE_PRESENT: your value` will apply the object
   and inspect the http log to check that the value in VALUE_PRESENT appears. The step will
-  wait ~ seconds for that value to show up.
\ No newline at end of file
+  wait ~ seconds for that value to show up.
+
+* Setting `WRITE-KUBE-OBJECT: false` will not export the KRM objects of the KCC resource.
```

Comment:
nit: To be more consist with other flags, maybe change the name to `TEST: NOT-EXPORT-KRM-OBJECT` or something similar?

Reply:
I think the idea is that this modifier can be added to any `TEST` ?

Reply:
Yes I am trying to experiment with this to see if there is value in it. I can come back and remove this if we are/ I am not getting the right mileage from excluding objects.